src/settings/Settings.js

```diff
@@ -394,4 +394,11 @@ export const SETTINGS = {
         // This is a tri-state value, where `null` means "prompt the user".
         default: null,
     },
+    "sendReadReceipts": {
+        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
```

Comment:
is it a oneliner to support granularizing this per-room (even with custom state events)?  If so, let's do it - my common case would be "i don't want my RRs to be seen in this room" rather than "i don't want my RRs to be seen globally"

Reply:
it's a couple lines, but can do. We don't have a labs section for rooms though, so it would be a secret setting for the time being.

it'd also do it through room account data so it doesn't need to be advertised to everyone.